ansible_builder/main.py

```diff
@@ -49,18 +54,39 @@ def build_command(self):
             self.build_context
         ]
 
+    def run_in_container(self, command, **kwargs):
+        wrapped_command = [
+            self.container_runtime, 'run',
+            '--rm', self.tag, '/bin/bash', '-c'
+        ] + [' '.join(command)]
+        return run_command(wrapped_command, **kwargs)
```

Comment:
I had to change this to the `-c` pattern so I could run more than 1 command with a single invocation.